app/styles/components/explore/side-bar.scss

```diff
@@ -24,3 +24,12 @@
   }
 }
 
+.location-input-container {
```

Comment:
This whole css is not needed. You just have to remove the class `action` from `ui action input` when there is no button.